templates/components/products/card.html

```diff
@@ -16,7 +16,35 @@
                 </div>
             {{/if}}
         {{/or}}
-        <a href="{{url}}" {{#if settings.data_tag_enabled}} data-event-type="product-click" {{/if}}>
+        <a href="{{url}}" 
+            aria-label="
+                {{name}},
+                {{#and price.price_range (if theme_settings.price_ranges '==' true)}}
+                    {{#and price.price_range.min.with_tax price.price_range.max.with_tax}}
+                        {{lang 'products.price_range_from'}} {{price.price_range.min.with_tax.formatted}} {{lang 'products.price_range_to'}} {{price.price_range.max.with_tax.formatted}}
+                    {{else}}
+                        {{lang 'products.price_range_from'}} {{price.price_range.min.without_tax.formatted}} {{lang 'products.price_range_to'}} {{price.price_range.max.without_tax.formatted}}
+                    {{/and}}
+                {{else}}
+                    {{#if price.with_tax}}
+                        {{#if price.non_sale_price_with_tax}}
+                            {{theme_settings.pdp-non-sale-price-label}}{{price.non_sale_price_with_tax.formatted}},
+                            {{theme_settings.pdp-sale-price-label}}{{price.with_tax.formatted}}
+                        {{else}}
+                            {{price.with_tax.formatted}}                        
+                        {{/if}}
+                    {{else}}
+                        {{#if price.non_sale_price_without_tax}}
+                            {{theme_settings.pdp-non-sale-price-label}}{{price.non_sale_price_without_tax.formatted}},
+                            {{theme_settings.pdp-sale-price-label}}{{price.without_tax.formatted}}
+                        {{else}}
+                            {{price.without_tax.formatted}}
+                        {{/if}}
+                    {{/if}}
+                {{/and}}
```

Comment:
In such case we are able to move this block to separate file

Reply:
done